Checkbox/Checkbox.module.less

```diff
@@ -23,7 +24,8 @@
 			height: @size;
 		}
 
-		&.selected {
+		&.selected,
+		&.indeterminate {
```

Comment:
I believe we don't have any guide for 'indeterminate' case, am I right?

Reply:
Yep, we don't have guide for 'indeterminate' case. I went for the simple solution and added a `minus` icon when `indeterminate` is true.

Reply:
👍